src/bindings/c/include/openvino/c/ov_compiled_model.h

```diff
@@ -28,34 +28,105 @@ typedef struct ov_compiled_model ov_compiled_model_t;
  */
 
 /**
- * @brief Gets runtime model information from a device.
+ * @brief Get the input size of ov_compiled_model_t.
  * @ingroup compiled_model
  * @param compiled_model A pointer to the ov_compiled_model_t.
- * @param model A pointer to the ov_model_t.
+ * @param input_size the compiled_model's input size.
+ * @return Status code of the operation: OK(0) for success.
+ */
+OPENVINO_C_API(ov_status_e) ov_compiled_model_inputs_size(const ov_compiled_model_t* compiled_model, size_t* size);
+
+/**
+ * @brief Get a const input port of ov_compiled_model_t.
```

Comment:
We need to provide information that this method works only in case of single input.